src/sentry/api/endpoints/event_file_committers.py

```diff
@@ -22,18 +21,10 @@ def get(self, request, project, event_id):
                                  retrieve (as reported by the raven client).
         :auth: required
         """
-
-        use_snuba = options.get('snuba.events-queries.enabled')
-
-        event_cls = event_cls = SnubaEvent if use_snuba else Event
-
-        event = event_cls.objects.from_event_id(event_id, project.id)
+        event = SnubaEvent.objects.from_event_id(event_id, project.id)
         if event is None:
             return Response({'detail': 'Event not found'}, status=404)
 
-        # populate event data
-        Event.objects.bind_nodes([event], 'data')
```

Comment:
Are you sure we can just remove this entire step for snuba events?

Reply:
I think this is covered by https://github.com/getsentry/sentry/blob/master/src/sentry/models/event.py#L506-L510 for Snuba events

Reply:
I am not sure this is the same. bind_nodes loads the node from nodestore explicitly. Did you figure out how the snuba event forces the nodestore load when accessing the 'data' field.

Reply:
Sorry you are right, we still need this. Putting it back.